doc/src/manual/ranch_ssl.asciidoc

```diff
@@ -156,6 +157,13 @@ fail_if_no_peer_cert (false)::
 Whether to refuse the connection if the client sends an
 empty certificate.
 
+handshake (full)::
+
+If `hello` is specified for this option, the handshake is
+paused after receiving the client hello message. The handshake
+can then be resumed via `handshake_continue/3`, or cancelled
+via `handshake_cancel/1`.
```

Comment:
Probably worth noting that this option can not be given to the handshake function.